### PR TITLE
Try skipping caching on macos

### PR DESCRIPTION
**Pull request by coolreader18**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -108,12 +110,21 @@ jobs:
           RUSTPYTHONPATH: ${{ github.workspace }}/Lib
         if: runner.os == 'Linux'
       - name: run cpython tests (macOS lightweight)
-        run: target/release/rustpython -m test -x test_argparse -x test_json -x test_bytes -x test_long -v
+        run:
+          target/release/rustpython -m test -v -x
+            test_argparse test_json test_bytes test_bytearray test_long test_unicode test_array
+            test_asyncgen test_list test_complex test_json test_set test_dis test_calendar
+            test_importlib
```
Comment (youknowone) [inline .github/workflows/ci.yaml:117]: importlib depends on filesystem. I think this is better to be not skipped.

**Review by youknowone — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (coolreader18) [reply .github/workflows/ci.yaml:117]: Should I unskip it for windows as well?

Comment (youknowone) [reply .github/workflows/ci.yaml:117]: I think so if possible. The winodws tests with `-x` were failing at the time I added it.

**Comment by coolreader18** [comment]
@youknowone I wasn't able to get windows test_importlib working; it completely succeeds on my machine, but on ci there's something weird with rmtree failing because it wasn't able to remove all the files in a directory, I think? Which might be because there's still an open handle to one of the files for some reason, and windows can't unlink a file until the handle's closed. Also, it's always only *1* of the files that doesn't get removed, for some reason. e.g.:

```
test_lacking_parent (test.test_importlib.source.test_file_loader.SimpleTest) ... rmtree unlinking dir D:\a\RustPython\RustPython\tmp\tmpq16syujf\_pkg
rmtree unlinking D:\a\RustPython\RustPython\tmp\tmpq16syujf\_pkg\mod.py
rmtree unlinking D:\a\RustPython\RustPython\tmp\tmpq16syujf\_pkg\__init__.py
rmtree unlinking dir D:\a\RustPython\RustPython\tmp\tmpq16syujf\_pkg\__pycache__
rmtree unlinking D:\a\RustPython\RustPython\tmp\tmpq16syujf\_pkg\__pycache__\mod.rustpython-01.pyc
ooh there's still ['mod.py']
D:\a\RustPython\RustPython\vm\pylib-crate\Lib\test\support\__init__.py:414: RuntimeWarning: tests may fail, delete still pending for D:\a\RustPython\RustPython\tmp\tmpq16syujf\_pkg
  _waitfor(_rmtree_inner, path, waitall=True)
OSError: The directory is not empty. (os error 145)
re-run rmdir('D:\\a\\RustPython\\RustPython\\tmp\\tmpq16syujf\\_pkg',)
ERROR
```

[ci run](https://github.com/RustPython/RustPython/runs/1345406722?check_suite_focus=true)

**Review by youknowone — APPROVED** [review]

**Pull request merged**